examples/vue-example-app/src/components/MapLayout.vue

```diff
@@ -19,6 +19,21 @@
     name: 'MapLayout',
     components: {
       AddressSearchMap
+    },
+    data() {
+      // Please make sure env variable VUE_APP_COVID_MAP_RADIUS is set.
+      const radius = parseInt(process.env.VUE_APP_COVID_MAP_RADIUS)
+      let radiusLabel
+      if (isNaN(radius)) {
+        radiusLabel = `We have a glitch in our system. We are aware of the issue. We appreciate your patience while we're looking into it.`
```

Comment:
Hey @anthony-malkoun I put this error message here in case there is no env var value.